src/better_lane_follow/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package>
+  <name>better_lane_follow</name>

```

Comment:
if we change the package name as per my comment above, will probably have to change it here as well